### ISSUE TITLE
Windows 10: Crash on startup when recent files were deleted since last launch

### ISSUE THREAD

**Issue by 8byr0** [bjorn/tiled#2032]
I'm running Tiled on Windows 10(version 1803) and I'm expecting an issue for a week.

## Reproduce
- Open editor
- Create a new project
- Load a tileset
- (you can do both previous multiple times)
- Close editor
- Delete the sources of the projects you created
- Try to launch  editor
- If everything went well it should crash

To make the app run again, run as administrator and you're all set.

## Investigations
After cloning this repository, I guess this is due to the keys contained in `Computer\HKEY_CURRENT_USER\Software\mapeditor.org\Tiled\recentFiles` that refer to missing files.

Are the files in these keys checked before launching the soft? It sounds like a null pointer exception...
Is there a way to access the app logs? I've been checking in AppData and ProgramData but could not find anything ! (also checked default QT locations...).

**Comment by bjorn** [comment]
Normally it pops up a "File not found" dialog for previously open files that are missing.

Could it be that you've tried to use the Worlds feature? I've fixed a crash that happens when a world file failed to load in the latest snapshot (and soon to be in Tiled 1.2.1). In that case please let me know if it still happens again after upgrading.

If it isn't that, is there any additional information you can provide?

**Comment by 8byr0** [comment]
Thanks for your reply, I'm currently running 1.2.0 so I'll try your new release ! 
I cannot remember if I had been using Worlds feature but it is not impossible, as long as I was handling the app (so I tried many functionalities) !

**Comment by bjorn** [comment]
You can try Tiled 1.2.1 now by downloading it from the [releases page](https://github.com/bjorn/tiled/releases/tag/v1.2.1).

**Comment by bjorn** [comment]
> I apologize, I ran into other projects since last time. I've been able to run tiled again by using 32 bits version instead of 64! But I have no further details about the issue's cause.

Hmm, so the crash on startup re-appears when you switch back to the 64-bit version of Tiled?

Note that switching between 32-bit and 64-bit may cause trouble in installer, so in this case I'd recommend to do a manual uninstall (of all installed versions of Tiled) before installing the other version.

**Comment by 8byr0** [comment]
Yes, it re-appears if I reinstall 64 bits version. What I actually did was to fully remove the 64bit version but if I reinstall it then it results in the same troubles as described previously !
